dali/pipeline/operator/builtin/make_contiguous.cu

```diff
@@ -16,12 +16,68 @@
 
 namespace dali {
 
-DALI_REGISTER_OPERATOR(MakeContiguous, MakeContiguous, Mixed);
+void MakeContiguousMixed::Run(MixedWorkspace &ws) {
+  const auto& input = ws.Input<CPUBackend>(0, 0);
+  int sample_dim = input.shape().sample_dim();
+  TensorListShape<> output_shape(batch_size_, sample_dim);
+  TensorLayout layout = input.GetLayout();
+  TypeInfo type = input.type();
+  size_t total_bytes = 0;
+  for (int i = 0; i < batch_size_; ++i) {
```

Comment:
Suggestion, do not rely on batch_size_ member since we have tentative plans to remove it. You can get the number of samples from the input shape

Reply:
Done